Include/internal/pycore_object.h

```diff
@@ -454,10 +455,6 @@ extern int _PyObject_IsAbstract(PyObject *);
 extern int _PyObject_GetMethod(PyObject *obj, PyObject *name, PyObject **method);
 extern PyObject* _PyObject_NextNotImplemented(PyObject *);
 
-// Pickle support.
```

Comment:
I think the CI failures is happening because of this removal.

Reply:
I guess I didn't merge correctly.